core/tests/protractor_utils/workflow.js

```diff
@@ -207,11 +218,15 @@ var createAndPublishTwoCardExploration = async function(
 
 // Here, 'roleName' is the user-visible form of the role name (e.g. 'Manager').
 var _addExplorationRole = async function(roleName, username) {
-  await element(by.css('.protractor-test-edit-roles')).click();
-  await element(by.css('.protractor-test-role-username')).sendKeys(username);
-  await element(by.css('.protractor-test-role-select')).
-    element(by.cssContainingText('option', roleName)).click();
-  await element(by.css('.protractor-test-save-role')).click();
+  var testEditRoles = element(by.css('.protractor-test-edit-roles'));
+  await action.click('testEditRoles', testEditRoles);
+  var testRoleUsername = element(by.css('.protractor-test-role-username'));
+  await action.sendKeys('testRoleUsername', testRoleUsername, username);
+  var testRoleSelect = element(by.css('.protractor-test-role-select'));
+  await action.click('testRoleSelect', testRoleSelect.element(
+    by.cssContainingText('option', roleName)));
```

Comment:
Could the description here indicate that it's the option under `testRoleSelect`? Maybe `testRoleSelectOption`?